submissions/evgeniy241984/html-css-popup/index.html

```diff
@@ -107,7 +107,7 @@
                         <label for="more__btn" class="more__label">
                             <span class="more__btn">More</span>
                         </label>
-                        <hr class="popup__divider">
+                        <span class="popup__divider"></span>
```

Comment:
Avoid creating and using HTML elements for decorative things.
In this situation better replace `span` by `border-top` of `.popup__menu-continue` block.

Reply:
All done! 
Please see changes in new commit! 
Thank you for th patience!)